Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -130,10 +144,16 @@ final class PledgeCTAContainerView: UIView {
         self?.animateView(self?.pledgeCTAButton, isHidden: isHidden)
       }
 
+    self.viewModel.outputs.retryStackViewIsHidden
+      .observeForUI()
+      .observeValues { [weak self] isHidden in
+        self?.updateRetryStackViewIsHidden(isHidden)
```

Comment:
This is a work-around for the fact that the height of the `retryStackView` is *usually* (depending on the device) larger than the height of the other views in the pledge CTA view (manage button, "Back this project" button etc.). Because the height was larger, simply hiding the `retryStackView` meant that the pledge CTA view appeared to be too tall for the "Back this project" button, because the parent view is accommodating for the height of the hidden `retryStackView`. Another alternative to this approach would be to only set the text of the `retryDescriptionLabel` when the retry stack view is being shown – it didn't feel like a worthwhile enough reason to add more complexity to the view model, so I went with this approach instead. Open to comments in case I missed something.

PS. iterating through the `retryStackView`'s subviews and hiding them also worked, but this seemed like a cleaner approach 🤷‍♀

Reply:
Is there a reason you chose to add the `retryStackView` with Auto Layout and not just to the `rootStackView`. I think if it were a child of a stack view then hiding/showing it should have the desired behaviour.

Reply:
🤔 let me try that

Reply:
That worked 🌟 brilliant